docs/source/advanced_usage.rst

```diff
@@ -46,8 +46,8 @@ auth-related HTTP requests.
 
 .. note::
     QVMs process incoming requests in parallel, while QPUs process them sequentially. If you encounter timeouts while
```

Comment:
Corrected ( as long as we're making fixes) :

> The QVM processes incoming requests in parallel, while a QPU may process them sequentially or in parallel (depending on the qubits used).

Reply:
Will do, thanks!